<launch>
    <!-- Argument fÃ¼r die Liste der Roboter -->
    <arg name="robot_names" default="[]" />
    <arg name="UR_prefixes" default="['UR10_l', 'UR10_r']" />

    <node pkg="handling" type="enable_all_URs.py" name="enable_all_URs" output="screen">
        <rosparam param="robot_names" subst_value="True">$(arg robot_names) </rosparam>
        <rosparam param="UR_prefixes" subst_value="True">$(arg UR_prefixes) </rosparam>
    </node>
</launch>